Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -757,135 +618,134 @@ protected override void OnRemoved()
+                "period",
+                "slash",
+                "backslash",
+                "leftbracket",
+                "rightbracket",
+                "minus",
+                "equals",
+                "a",
+                "b",
+                "c",
+                "d",
+                "e",
+                "f",
+                "g",
+                "h",
+                "i",
+                "j",
+                "k",
+                "l",
+                "m",
+                "n",
+                "o",
+                "p",
+                "q",
+                "r",
+                "s",
+                "t",
+                "u",
+                "v",
+                "w",
+                "x",
+                "y",
+                "z",
+                "1",
+                "2",
+                "3",
+                "4",
+                "5",
+                "6",
+                "7",
+                "8",
+                "9",
+                "0",
+                "leftshift",
+                "rightshift",
+                "leftalt",
+                "rightalt",
+                "leftctrl",
+                "rightctrl",
+                "leftmeta",
+                "rightmeta",
+                "contextmenu",
+                "escape",
+                "leftarrow",
+                "rightarrow",
+                "uparrow",
+                "downarrow",
+                "backspace",
+                "pagedown",
+                "pageup",
+                "home",
+                "end",
+                "insert",
+                "delete",
+                "capslock",
+                "numlock",
+                "printscreen",
+                "scrolllock",
+                "pause",
+                "numpadenter",
+                "numpaddivide",
+                "numpadmultiply",
+                "numpadplus",
+                "numpadminus",
+                "numpadperiod",
+                "numpadequals",
+                "numpad0",
+                "numpad1",
+                "numpad2",
+                "numpad3",
+                "numpad4",
+                "numpad5",
+                "numpad6",
+                "numpad7",
+                "numpad8",
+                "numpad9",
+                "f1",
+                "f2",
+                "f3",
+                "f4",
+                "f5",
+                "f6",
+                "f7",
+                "f8",
+                "f9",
+                "f10",
+                "f11",
+                "f12",
+                "oem1",
+                "oem2",
+                "oem3",
+                "oem4",
+                "oem5",
+            };
+            m_Keys = new KeyControl[keyStrings.Length];
+            for (int i = 1; i < keyStrings.Length; i++)
+            {
+                m_Keys[i] = builder.GetControl<KeyControl>(keyStrings[i]);
+                m_Keys[i].keyCode = (Key)i;
+            }
+            Debug.Assert(keyStrings[(int)Key.OEM5] == "oem5");
```

Comment:
Always use the message arg on Debug.Assert. I made the mistake early on of not doing so consistently and then found it's often really hard to track down the asserts as unlike in native, it does not print the expression when an assert fails.